app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt

```diff
@@ -401,28 +397,23 @@ class MainActivity : AppCompatActivity() {
                         editor.putString(Constants.WEATHER_RESPONSE_DATA, weatherResponseJsonString)
                         editor.apply()
 
-//                        var location : String = weatherList.sys.country
-//                        if (location == "PH") {
+                        var location : String = weatherList.sys.country
+                        if (location == "PH") {
```

Comment:
why do you filter other contries except PH?

Reply:
I tried to filter City name valid for Philippines only. I forgot to remove the Code before I commit.